test.rb

```diff
@@ -0,0 +1,21 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
```

Comment:
__[rubocop]__
* Use `!` instead of `not`. [:ref](https://github.com/bbatsov/ruby-style-guide#bang-not-not)
* Do not leave space between `!` and its argument. [:ref](https://github.com/bbatsov/ruby-style-guide#no-space-bang)